codebuild/bin/install_ubuntu_dependencies.sh

```diff
@@ -25,7 +25,7 @@ apt-key list
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
 sudo apt-get update -o Acquire::CompressionTypes::Order::=gz
 
-DEPENDENCIES="unzip make indent kwstyle libssl-dev tcpdump valgrind lcov m4 nettle-dev nettle-bin pkg-config gcc g++ zlibc zlib1g-dev python3-pip llvm curl git"
+DEPENDENCIES="unzip make indent kwstyle libssl-dev tcpdump valgrind lcov m4 nettle-dev nettle-bin pkg-config gcc g++ zlibc zlib1g-dev python3-pip llvm curl git tox"
```

Comment:
For CodeBuild, tox should also be added to [buildspec_ubuntu.yml](https://github.com/awslabs/s2n/blob/master/codebuild/spec/buildspec_ubuntu.yml#L40)

Reply:
Added.